strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/ArtifactManagementService.java

```diff
@@ -153,9 +155,13 @@ public long store(RepositoryPath repositoryPath,
         {
             repositoryPathLock.unlock(repositoryPath);
         }
+        
+        result.getEvents().stream().forEach(Runnable::run);
```

Comment:
` result.getEvents().stream().forEach(Runnable::run);`

we should use ` @TransactionalEventListener` instead of this custom workarounds

Reply:
good suggestion, I like it
thanks

Reply:
👍

Reply:
@sbespalov : Would you like to look into this in a follow-up pull?

Reply:
yeah, transactional events already implemented in this PR, I just forget to remove this `Runnable` part, will do it in follow up PR.